templates/zerver/help/scim.md

```diff
@@ -1,62 +1,88 @@
 # SCIM provisioning
 
-SCIM (System for Cross-domain Identity Management) is an standard
+{!admin-only.md!}
+
+SCIM (System for Cross-domain Identity Management) is a standard
 protocol used by Single Sign-On (SSO) services and identity providers
-to provision/deprovision user accounts and groups. Zulip's SCIM
-integration is currently beta and has a few limitations:
+to provision/deprovision user accounts and groups.
+
+Zulip supports SCIM integration, both for Zulip Cloud and self-hosted Zulip
+servers. This page describes how to configure SCIM in Okta.
+
+If you are [self-hosting](/self-hosting/) Zulip, these instructions can still
+be used to set up your SCIM deployment with Okta. The documentation on this page
+may also be a useful reference for how to set up specific SCIM providers.
+
+Zulip's SCIM integration is currently beta and has the following limitations:
 
```

Comment:
Hmm, not really, I don't think these limitations make it a beta. That wording was originally added since this was a brand new feature with very little production testing. Now that it has been running mostly successfully  with Okta for a while with a couple of bugfixes, it might be right to remove the wording. @timabbott may have some thoughts here too